.travis.yml

```diff
@@ -76,7 +76,7 @@ install:
   - source bin/hunter_env.sh
   
 script:
-  - bin/drishti_build.sh "${TOOLCHAIN}" "${CONFIG}" "${INSTALL}"
+  - travis_wait 200 bin/drishti_build.sh "${TOOLCHAIN}" "${CONFIG}" "${INSTALL}"
```

Comment:
We will not be able to see logs but still will have timeouts...

Reply:
`travis_wait` hides the logs?

Reply:
Yes, I can't see anything: https://travis-ci.org/elucideye/drishti/jobs/289534264

Reply:
I see.  Sample output:

```
++echo -ne 'Still running (1 of 200): bin/drishti_build.sh' gcc-5-pic-hid-sections-lto Release '--strip\r'
++sleep 60
++'[' 1 -lt 200 ']'
++count=2
++echo -ne 'Still running (2 of 200): bin/drishti_build.sh' gcc-5-pic-hid-sections-lto Release '--strip\r'
++sleep 60
++'[' 2 -lt 200 ']'
++count=3
++echo -ne 'Still running (3 of 200): bin/drishti_build.sh' gcc-5-pic-hid-sections-lto Release '--strip\r'
++sleep 60
++'[' 3 -lt 200 ']'
```

Reply:
Does this buffer logs and dump to cout after success?

Reply:
https://github.com/travis-ci/worker/issues/283#issue-207659669

Reply:
> Does this buffer logs and dump to cout after success?

Yes, but if we hit timeout we will see nothing.

Reply:
https://github.com/travis-ci/travis-ci/issues/7323#issuecomment-298832640

Ok.  We can remove this in the next PR.  The last android timeout was due to the global threshold I believe.  

> The job exceeded the maximum time limit for jobs, and has been terminated.

(I haven't checked recent android toolchain timeouts to see if they are due to 10 minute stdout timeout or the global timeouts.)

Reply:
This is probably the 4th or 5th restart.  When the `travis_wait` is running it gives a direct measure of how long the build step has been running.  It seems things get to some point (like `count=38` or so) and then it just sits there as if the VM is frozen as the global clock continues:

```
++count=38
++echo -ne 'Still running (38 of 200): bin/drishti_build.sh' android-ndk-r10e-api-19-armeabi-v7a-neon-hid-sections MinSizeRel '--strip\r'
++sleep 60
```